themes-default/slim/views/history.mako

```diff
@@ -104,23 +108,23 @@ const startVue = () => {
         <div class="layout-controls pull-right">
             <div class="show-option">
                 <span>Limit:</span>
-                    <select name="history_limit" id="history_limit" class="form-control form-control-inline input-sm">
-                        <option value="10" ${'selected="selected"' if limit == 10 else ''}>10</option>
-                        <option value="25" ${'selected="selected"' if limit == 25 else ''}>25</option>
-                        <option value="50" ${'selected="selected"' if limit == 50 else ''}>50</option>
-                        <option value="100" ${'selected="selected"' if limit == 100 else ''}>100</option>
-                        <option value="250" ${'selected="selected"' if limit == 250 else ''}>250</option>
-                        <option value="500" ${'selected="selected"' if limit == 500 else ''}>500</option>
-                        <option value="750" ${'selected="selected"' if limit == 750 else ''}>750</option>
-                        <option value="1000" ${'selected="selected"' if limit == 1000 else ''}>1000</option>
-                        <option value="0"   ${'selected="selected"' if limit == 0   else ''}>All</option>
-                    </select>
+                <select v-model="limit" name="history_limit" id="history_limit" class="form-control form-control-inline input-sm">
+                    <option value="10">10</option>
```

Comment:
I don't believe these would work, because `limit` is an integer but these values are parsed as strings.

Reply:
It worked fine for all of the other number based selects so it should be fine here.

Reply:
Fixed.